example/example.js

```diff
@@ -17,6 +17,16 @@ const config = {
   }
 };
 
-app.use('/', SensorThings(config));
+const clientHooks = {
+  'Things': {
+    'beforeCreate': (instance, options) => {
```

Comment:
I have organized the hooks by Entity because using global hooks I wasn't able to determine on which Entity the hook was operating on.